indexer/etl/l1_etl.go

```diff
@@ -20,43 +19,49 @@ type L1ETL struct {
 	db *database.DB
 }
 
-func NewL1ETL(log log.Logger, db *database.DB, client node.EthClient, contracts config.L1Contracts) (*L1ETL, error) {
+// NewL1ETL creates a new L1ETL instance that will start indexing from different starting points
```

Comment:
/s/start/startingHeight

Reply:
Deferring to `startHeight` if that's cool

Reply:
[dd4f561](https://github.com/ethereum-optimism/optimism/pull/6914/commits/dd4f561552f16d8de1e09cc46f39079d2bdcb77d)